paimon-flink/paimon-flink-cdc/src/main/java/org/apache/paimon/flink/action/cdc/pulsar/PulsarActionUtils.java

```diff
@@ -324,8 +324,7 @@ static DataFormat getDataFormat(Configuration pulsarConfig) {
     /** Referenced to {@link PulsarPartitionSplitReader#createPulsarConsumer}. */
     static MessageQueueSchemaUtils.ConsumerWrapper createPulsarConsumer(
             Configuration pulsarConfig, String topic) throws PulsarClientException {
-        SourceConfiguration pulsarSourceConfiguration =
-                toSourceConfiguration(preprocessPulsarConfig(pulsarConfig));
```

Comment:
Why modify this?

Reply:
I found that the `preprocessPulsarConfig` method will be executed first in the `toSourceConfiguration` method, so preprocessPulsarConfig is deleted here.

```java
    static SourceConfiguration toSourceConfiguration(Configuration rawConfig) {
        return new SourceConfiguration(preprocessPulsarConfig(rawConfig));
    }
```